unit/solvers/refinement/string_constraint_instantiation/instantiate_not_contains.cpp

```diff
@@ -188,8 +188,7 @@ SCENARIO("instantiate_not_contains",
     // Generating the corresponding axioms and simplifying, recording info
     symbol_tablet symtab;
     const namespacet empty_ns(symtab);
-    string_constraint_generatort::infot info;
-    string_constraint_generatort generator(info, ns);
+    string_constraint_generatort generator(ns);
```

Comment:
If this is a change necessitated by one of the earlier commits, please merge it. But it might also just be cleanup, don't really know.

Reply:
squashed